app/utils/timezone.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import DeviceInfo from 'react-native-device-info';
+import {updateUser} from 'app/actions/views/edit_profile';
+import {mattermostStore} from 'app/mattermost';
```

Comment:
I really don't like this is there another way you can access the store or even the dispatch, getState from the store?

create an action instead?